src/components/form-elements/_form-elements.scss

```diff
@@ -95,30 +93,27 @@ $includeHtml: false !default;
       }
     }
 
-    &:before {
-      $iconSize: $selectFontSize + $iconBoostValue;
-      font-size: $iconSize;
-      font-weight: bold;
+    &__icon {
       position: absolute;
-      right: $selectHeight / 2 - $iconSize / 2;
-      top: 50%;
+      right: ($selectHeight / 4 + rhythm(0.334)) - $selectIconSize / 2;
```

Comment:
Should I change all 0.334 to 0.333 ?
Maybe we can have simple variables like $1of3 = 1 / 3;  ?